saltyrtc/server/server.py

```diff
@@ -951,7 +951,7 @@ def clean(self, path: Path) -> None:
 class Server:
     subprotocols = [
         SubProtocol.saltyrtc_v1.value
-    ]  # type: ClassVar[Sequence[SubProtocol]]
+    ]  # type: ClassVar[Sequence[str]]
```

Comment:
Note: I went for the quickfix here by changing the type to str. This could also be solved by changing all uses of `Server.subprotocols` to `[sp.value for sp in Server.subprotocols]`, but I don't know the codebase well and was a bit afraid to break something. If the only usage is on line 181, that would be an easy fix though.

Reply:
Maybe add a _todo_ comment above

Reply:
Done in #123.